### ISSUE TITLE
Make it easier to join bridged rooms

### ISSUE THREAD

**Issue by mcatanzaro** [matrix-org/matrix-appservice-irc#981]
Here is what I see when trying to join a bridged GIMPNet room on https://gnome.riot.im:

![Screenshot from 2020-02-14 10-47-46](https://user-images.githubusercontent.com/1424966/74551362-aa1c1d00-4f18-11ea-8a62-54be75ce54e4.png)

Note that there is only one bridged room visible. To join any other room, I have to type the name of the room manually. I also have to type a # at the start of the name, or joining the room will not work.

Matrix should display a full list of available IRC rooms, like an IRC client would do. Ideally it would also not require the leading #.

**Comment by Half-Shot** [comment]
Yep, this requires a change in lots of places to implement. Notably Riot needs to support querying the server for multiple 'locations', and the bridge needs to support the 3pid api to send them. I'll mark it as a feature request.
